packages/panna-sdk/src/core/utils/activity_mockedCache.test.ts

```diff
@@ -549,7 +549,7 @@ describe('updateInternalTransactionsCache', () => {
       .mockImplementation(() => mockCacheInstanceLocal);
 
     const mockRequestResponse = {
-      items: [{}],
+      items: [],
```

Comment:
The previous comment about leaving empty objects to validate the cache behavior is also applicable to this and the remaining testcases below :)

Reply:
It doesn't work. should we skip this test again to avoid spending too much time now?

Reply:
Yes, let's revert the changes within the testcases then ;)